test/browser-compile/src/components/app-root/app-root.tsx

```diff
@@ -57,7 +57,6 @@ export class AppRoot {
   }
 
   async compile() {
-    console.clear();
```

Comment:
When running this app, the console gets cleared after the worker loads, so the traspile problems weren't immediately obvious when I opened dev tools. 

PS: We should figure out how to automate this with Karma.

Reply:
Ah, I see now. It was clearing debugging statements in this component itself

And yes! Can you please put an item on the refinement sheet to discuss next week?

Reply:
Done!